test/lib/config/sets-options.js

```
@@ -0,0 +1,157 @@
+                    }
+                }
+            });
+
+            assert.deepEqual(config.sets.someSet.files, ['some/path']);
+        });
+
+        it('should not accept non-string arrays', () => {
+            assert.throws(() => {
+                createConfig({
+                    sets: {
+                        someSet: {
+                            files: [100500]
+                        }
+                    }
+                });
+            }, Error);
+        });
+
+        it('should accept array with strings', () => {
+            const config = createConfig({
+                sets: {
+                    someSet: {
+                        files: [
+                            'some/path',
+                            'other/path'
+                        ]
+                    }
+                }
+            });
+
+            assert.deepEqual(config.sets.someSet.files, [
+                'some/path',
+                'other/path'
+            ]);
+        });
+    });
+
+    describe('browsers', () => {
+        it('should contain all browsers by default', () => {
+            var config = createConfig({
+                browsers: {
+                    b1: {},
+                    b2: {}
+                },
+                sets: {
+                    someSet: {
+                        files: ['some/path']
+                    }
+                }
+            });
+
+            assert.deepEqual(config.sets.someSet.browsers, ['b1', 'b2']);
+        });
+
+        it('should not accept non-arrays', () => {
+            assert.throws(() => {
+                createConfig({
+                    sets: {
+                        someSet: {
+                            files: ['some/path'],
+                            browsers: 'something'
+                        }
+                    }
+                });
+            }, Error);
+        });
+
+        it('should not accept unknown browsers', () => {
```

should throw error if in set contains only unknown browsers